src/test/unittest/basetest.py

```diff
@@ -253,6 +254,22 @@ def _get_utenv(self):
             'UNITTEST_NUM': str(self.testnum)
         }
 
+    def _debug_log_env(self):
```

Comment:
This hardcodes the library names and the log level. I think the proper solution would be to have a base test type for each library that would set the appropriate env variables, and then tests would derive from the appropriate base type.
But that's a lot of changes...

Move the array of library names and the log level to someplace where won't forget it exists. testconfig.py maybe?